Formula/x/xnvme.rb

```diff
@@ -0,0 +1,69 @@
+class Xnvme < Formula
+  desc "Cross-platform libraries and tools for efficient I/O and low-level control"
+  homepage "https://xnvme.io/"
+  url "https://github.com/OpenMPDK/xNVMe/releases/download/v0.7.2/xnvme-0.7.2.tar.gz"
+  sha256 "1cb849b537cfddc15d82b8f4622fe3f999b4c7c0542c55b8d09b485e016e942e"
+  license "BSD-3-Clause"
+
+  depends_on "meson" => :build
+  depends_on "ninja" => :build
+  depends_on "python@3.12" => :build
+  depends_on "pkg-config"
```

Comment:
Is this needed at runtime?

Reply:
The xnvme formula provides library as well as cli-tools. To consume the library a pkg-config file is provided, to make proper use of that, then the pkg-config cli-tool is needed.

As an example, then the library-test shows how a user of the xNVMe library would use pkg-config to consume the library.